code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -473,3 +473,27 @@ GLOBAL_LIST_INIT(abductor_recipes, list ( \
 
 /obj/item/stack/sheet/mineral/coal/ten
 	amount = 10
+
+//Metal Hydrogen
+GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
+	new /datum/stack_recipe("incomplete servant golem shell", /obj/item/golem_shell/servant, req_amount=30, res_amount=1),
+	))
+
+/obj/item/stack/sheet/mineral/metal_hydrogen
+	name = "Metal Hydrogen"
+	icon_state = "sheet-metalhydrogen"
+	item_state = "sheet-metalhydrogen"
+	singular_name = "Metal Hydrogen sheet"
+	force = 5
+	throwforce = 5
+	w_class = WEIGHT_CLASS_NORMAL
+	resistance_flags = FIRE_PROOF | LAVA_PROOF | ACID_PROOF | INDESTRUCTIBLE
+	throw_speed = 1
+	throw_range = 3
+	point_value = 100
+	custom_materials = list(/datum/material/metalhydrogen=MINERAL_MATERIAL_AMOUNT)
+	merge_type = /obj/item/stack/sheet/mineral/metal_hydrogen
```

Comment:
You don't need to add values the parent already has (such as force = 5) again if they're duplicates.

Reply:
Ok I'll remove them in the next commit!